packages/react/src/SelectPanel/SelectPanel.docs.json

```diff
@@ -144,7 +144,43 @@
       "type": "string | React.ReactElement",
       "defaultValue": "null",
       "description": "Footer rendered at the end of the panel"
+    },
+    {
+      "name": "message",
+      "type": "React.ReactNode",
```

Comment:
I think the type above is still `React.ReactNode[]`, just wanted to point this out in case one of them needs to change 👀

Reply:
See https://github.com/primer/react/pull/5142#discussion_r2004718150